tests/test_github.py

```diff
@@ -25,17 +25,13 @@ def test_credfilename(githubenv):
 
 CREDFILE = """zappa
 45463104f006ccb3a512fb20e31b9a50f10ba38b
-55d60676
 """
 
 
 def test_read_credfile(githubenv):
     with tempfile.NamedTemporaryFile('w+t') as f:
         f.write(CREDFILE)
         f.flush()
-        username, token, ghid = read_credfile(f.name)
+        username, token = read_credfile(f.name)
```

Comment:
Does this mean that `read_credfile` only supports the newer format?
Will we need to regenerate all the credfiles?

I am making this comment solely in my personal capacity and am not conveying any rights to any intellectual property of any third parties.

Reply:
Thanks! yes, `read_credfile` only supports the newer format. The latest push should check if the credfile conforms to the newest format. I checks that the credfile only contains two lines (username and token), whereas the old version contains 3 lines. This might be fragile but short of crafty regex work it was what I came up with. 

Also the latest push should fix (circle) CI issues associated with Xonsh and one failing test.
Travis is still failing, now when it attempts to build the docs. I need to look into this.